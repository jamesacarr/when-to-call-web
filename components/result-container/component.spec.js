import React from 'react';
import { shallow } from 'enzyme';

import ResultContainer from './component';

describe('<ResultContainer/>', () => {
  it('renders correctly', () => {
    const classes = {
      root: 'root'
    };
    const result = { test: 'stuff' };

    const wrapper = shallow(<ResultContainer classes={classes} result={result}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
