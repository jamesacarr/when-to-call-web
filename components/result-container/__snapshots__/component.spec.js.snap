// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ResultContainer/> renders correctly 1`] = `
<WithStyles(GridWrapper)
  className="root"
  container={true}
>
  <WithStyles(GridWrapper)
    item={true}
    xs={12}
  >
    <WithStyles(MapContainer)
      onLoad={[Function]}
      result={
        Object {
          "test": "stuff",
        }
      }
    />
  </WithStyles(GridWrapper)>
</WithStyles(GridWrapper)>
`;
